### ISSUE TITLE
Shufflenet

### ISSUE THREAD

**Issue by mosheliv** [qubvel/classification_models#20]
Thank you for the amazing lib. would it be possible to add shufflenet to it?

**Comment by qubvel** [comment]
Hi @mosheliv 
Do you have an exmaples of shufflenet pretrained on keras/pytorch/mxnet?

**Comment by mosheliv** [comment]
I have used https://github.com/opconty/keras-shufflenetV2 (some of the other implementations were less stable)
I couldn't find pretrained weights for any keras implementation but in my case it wasn't that important.
pytorch, naturally, have much better pretrained models, like https://github.com/jaxony/ShuffleNet, https://github.com/ericsun99/Shufflenet-v2-Pytorch or https://github.com/miaow1988/ShuffleNet_V2_pytorch_caffe
